ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -29,7 +29,6 @@ cat >> "${CONFIG}" << EOF
 baseDomain: ${BASE_DOMAIN}
 platform:
   azure:
-    baseDomainResourceGroupName: os4-common
```

Comment:
How about just a simple replacement like this:
```
$(if [[ -z "${PUBLISH}" ]] || [[ "${PUBLISH}" == "External" ]]; then echo "baseDomainResourceGroupName: os4-common"; fi)
```

Reply:
I did not see any difference after you resolve this.

Reply:
Okay, I saw the minor update in the following lines, if you want a dedicated yq patch block to deal with it, I am okay with it too.